testdata/mod/github.com_gobin-testrepos_simple-main-directory-replace_v1.0.0.txt

```diff
@@ -0,0 +1,27 @@
+-- .mod --
+module github.com/gobin-testrepos/simple-main-directory-replace
+
+require github.com/gobin-testrepos/food v1.0.0
+
+replace github.com/gobin-testrepos/food => /path/to/nowhere
```

Comment:
s/path/road/
? 😁

more seriously, it might be slightly better to have more than one replace statement, including a non-directory replace, just to exercise the logic a bit more.

Reply:
Ha! Great suggestion, will make that one.

We already have a non-directory replace test; we were, in effect, missing a directory based one before 😄 

https://github.com/myitcv/gobin/blob/master/testdata/replace.txt